extractors/register_parser.py

```
@@ -2,28 +2,45 @@
-        json_path = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS")
-        if not json_path or not os.path.exists(json_path):
-            raise RuntimeError(
-                "환경 변수 GOOGLE_APPLICATION_CREDENTIALS가 없거나 경로가 잘못되었습니다.")
+        
+        # 상대 경로를 절대 경로로 변환
+        try:
+            json_path = get_google_credentials_path()
+            # 환경 변수를 절대 경로로 업데이트
+            os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = json_path
+        except (ValueError, FileNotFoundError) as e:
+            raise RuntimeError(f"Google Cloud 인증 설정 오류: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)